internal/storage/dbmodel/instance.go

```diff
@@ -35,6 +35,7 @@ type InstanceFilter struct {
 	States                       []InstanceState
 	Expired                      *bool
 	DeletionAttempted            *bool
+	BindingExists                *bool
```

Comment:
We can have more than 1 binding per instance? 
So the meaning is `is there at least one binding`, correct?

Reply:
yes